File: src/dataset/cpp/raindrops_exercism-nenjotsu/claude/ClaudeSonnet4_raindrops_v3.cpp
#include "raindrops.h"

namespace raindrops {
    std::string convert(int n) {
        std::string result;
        result.reserve(15);
        
        if (n % 3 == 0) result += "Pling";
        if (n % 5 == 0) result += "Plang";
        if (n % 7 == 0) result += "Plong";
        
        return result.empty() ? std::to_string(n) : result;
    }
}

File: src/dataset/javascript/raindrops_exercism-elenamountz/claude/ClaudeSonnet4_raindrops_v3.js
class Raindrops {
  convert(number) {
    const factors = [
      [3, 'Pling'],
      [5, 'Plang'],
      [7, 'Plong']
    ];
    
    let output = '';
    for (const [factor, sound] of factors) {
      if (number % factor === 0) {
        output += sound;
      }
    }
    
    return output || number.toString();
  }
}

export default Raindrops;